### PR TITLE
Add labels tuning curves

### LINKED ISSUE #536: label on response curves / tuning curve plot axes
just noticed there are no labels, i.e. firing rate and input current / x

### PR DESCRIPTION
**Pull request by Seanny123**
Resolves #536

### CODE REVIEW

File: nengo_gui/static/modal.js [hunk 1]
```diff
@@ -408,9 +417,31 @@ Nengo.Modal.prototype.multiline_plot = function(selector, x, ys) {
         .orient("left");
     graph.append("g")
         .attr("class", "axis axis_y unselectable")
-        .attr("transform", "translate(" + m.left + ",0)")
+        .attr("transform", "translate(" + margin.left + ",0)")
         .call(yAxisLeft);
 
+    // label the axes
+    if(x_label !== ""){
```
Comment (tcstewar) [inline nengo_gui/static/modal.js]: space after ```if``` and before ```{```

File: nengo_gui/static/modal.js [hunk 2]
```diff
@@ -408,9 +417,31 @@ Nengo.Modal.prototype.multiline_plot = function(selector, x, ys) {
...
+        svg.append("text")
+            .attr("class", "x label")
+            .attr("text-anchor", "middle")
+            .attr("x", graph_w / 2)
+            .attr("y", text_offset + graph_h - margin.bottom / 2)
+            .text(x_label);
+    }
+
+    if(y_label !== ""){
```
Comment (tcstewar) [inline nengo_gui/static/modal.js]: space after ```if``` and before ```{```

**Comment by tcstewar** [comment]
Looks really good, other than four tiny little missing spaces!  :)

**Comment by Seanny123** [comment]
Added missing spaces

**Pull request merged**